tests/geoip/suricata.yaml

```diff
@@ -0,0 +1,13 @@
+%YAML 1.1
+---
+
+geoip-database: /var/lib/GeoIP/GeoLite2-Country.mmdb
```

Comment:
I don't think we can assume all OSs will put it here.

Reply:
@jasonish any idea on how to to this cross OSes ?

Reply:
> @jasonish any idea on how to to this cross OSes ?

Unfortunately not. And this is not the location used on CentOS/Fedora, so we can't even settle on a location across the most popular distros.

Maybe a pre-script that hunts it down in various known locations and copies it into a temporary directory, somehow skipping the test if it can't be found? This would require some extra support to suricata-verify I think.

Reply:
It looks like we can build our own maxmindb easily. Which means we could have a small one embedded in the tests. I'm gonna try to create one.

Reply:
Got a success on that. One last thing to fix and gonna open a new PR.